src/bp-messages/bp-messages-notifications.php

```diff
@@ -23,7 +23,7 @@
  *                                  notifications; 'array' for WP Toolbar. Default: 'string'.
```

Comment:
Need to add two param in doc comment
$notification_id
$screen

also check
$return = apply_filters( "bp_messages_{$action}_notification", $return, $item_id, $secondary_item_id, $total_items, $format, $notification_id, $screen );